tests/test_tng.py

```diff
@@ -1,6 +1,6 @@
 import pytng
-
```

Comment:
This is a big test file ... could be split into more manageable logical units.

Reply:
split by example file could be a good option, or split by API segments ie static properties vs volatile trajectory data?

Reply:
Split by API elements seems more logical than by test files unless one file are more "integration tests" (that look at the whole thing and check that everything works together) while others are more "unit tests" that check for specific functionality.

Reply:
Ack :+1: